app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -113,48 +111,15 @@ def analytics(user: AnonymousUser.new)
     Analytics.new(user: user, request: nil, session: {}, sp: nil)
   end
 
-  def email_analytics_attributes(enrollment)
-    {
-      enrollment_code: enrollment.enrollment_code,
-      timestamp: Time.zone.now,
-      service_provider: enrollment.issuer,
-      wait_until: mail_delivery_params(enrollment.proofed_at)[:wait_until],
-    }
-  end
-
-  def enrollment_analytics_attributes(enrollment, complete:)
-    {
-      enrollment_code: enrollment.enrollment_code,
-      enrollment_id: enrollment.id,
-      minutes_since_last_status_check: enrollment.minutes_since_last_status_check,
-      minutes_since_last_status_check_completed:
-        enrollment.minutes_since_last_status_check_completed,
-      minutes_since_last_status_update: enrollment.minutes_since_last_status_update,
-      minutes_since_established: enrollment.minutes_since_established,
-      minutes_to_completion: complete ? enrollment.minutes_since_established : nil,
-      issuer: enrollment.issuer,
-    }
-  end
-
-  def response_analytics_attributes(response)
-    return { response_present: false } unless response.present?
-
-    {
-      fraud_suspected: response['fraudSuspected'],
-      primary_id_type: response['primaryIdType'],
-      secondary_id_type: response['secondaryIdType'],
-      failure_reason: response['failureReason'],
-      transaction_end_date_time: parse_usps_timestamp(response['transactionEndDateTime']),
-      transaction_start_date_time: parse_usps_timestamp(response['transactionStartDateTime']),
-      status: response['status'],
-      assurance_level: response['assuranceLevel'],
-      proofing_post_office: response['proofingPostOffice'],
-      proofing_city: response['proofingCity'],
-      proofing_state: response['proofingState'],
-      scan_count: response['scanCount'],
-      response_message: response['responseMessage'],
-      response_present: true,
-    }
+  def percent_errored
+    error_rate = 0
+    if enrollment_outcomes[:enrollments_checked] > 0
+      error_rate =
+        (enrollment_outcomes[:enrollments_errored].fdiv(
+          enrollment_outcomes[:enrollments_checked],
+        ) * 100).round(0)
```

Comment:
Why are we changing from `round(2)` to `round(0)`?

Reply:
The `percent_enrollments_errored` is set to have no decimals in the [spec](https://github.com/18F/identity-idp/blob/a3761656177f7748047ff11593f873be00d8013e/spec/jobs/get_usps_proofing_results_job_spec.rb#L345).

I can change the spec to expect 2 decimal precision.